cluster/juju/layers/kube-dns/config.yaml

```diff
@@ -0,0 +1,9 @@
+options:
+  domain:
+    type: string
+    default: "cluster.local"
```

Comment:
Does this need to be a list? In some of the kubernetes configuration scripts I see them setting the sans to different DNS names:
```
DNS:kubernetes,DNS:kubernetes.default,DNS:kubernetes.default.svc,DNS:kubernetes.default.svc.cluster.local
```